cookbook/default-snippets.rst

```diff
@@ -0,0 +1,32 @@
+How to define a default-snippet?
+================================
+
+Sulu gives the content-manager an easy way to define default snippets per type.
+This default-snippets automatically be used if the snippet-selection is empty on
+a specific page. Be aware of that this "default" selection will not be displayed
+in the form and can have a big impact on the page.
+
+.. note::
+
+    This feature is disabled by default and can be activated by setting the
+    config value `sulu_snippet.types.snippet.fallback_enabled` to true.
+
+When the defaults will be used?
+-------------------------------
+
+There have to be conditions which have to match:
+
+1. The feature has to be activated
+2. The snippet-selection parameter ``snippetType`` and ``default`` has to be
+   defined. See :doc:`../reference/content-types/snippet`.
+3. The default snippet for the configured type has to be selected in the
+   webspace settings.
+
+If this conditions match the default snippet will be injected to the page. The
+template of the page does not have to be changed.
```

Comment:
Does this last sentence make any sense? Could anybody have the idea that the template should be changed? I think this sentence confuses more than it helps 😕